ui/src/tool-bar/tool-bar.tsx

```diff
@@ -471,8 +472,8 @@ export const ToolBar: React.FC<ToolbarParams> = ({
                                                 iconPosition={ButtonIconPosition.END}
                                                 disabled={!!splittingState}
                                                 inheritTypography={true}
-                                                onClick={(e) => {
-                                                    e.stopPropagation(); // otherwise layout selection gets the click
+                                                ref={designIdButtonRef}
+                                                onClick={() => {
```

Comment:
🍖 

Täällä oli siis aiemmin `stopPropagation()`-kutsu sen takia, että tämä klikkihandlerin ajautuminen aiheutti valinta-dropdownin (joka leijuu absoluuttisesti positioituna ja melkein bodyyn saakka portaloituna `CloseableModal`:in sisällä) tulemisen näkyviin (haluttua). Tuon vastaesiintuodun `CloseableModal`:in globaali klikkikäsittelijä sitten huomasi tämän ja käsitteli klikin, huomasi että sehän tuli modalin ulkopuolelta ja sulki modalin. Great success! Nyt tämä on hoidettu muilla mekanismeilla.